webapp/src/components/AdminTable.js

```diff
@@ -2,49 +2,47 @@ import React from "react";
 import MaterialTable from "@material-table/core";
 import { ThemeProvider, createTheme } from "@mui/material";
 
-import "../components/css/Admin.css";
+import "../features/adminRsuTab/Admin.css";
 
 const AdminTable = (props) => {
   const defaultMaterialTheme = createTheme({

```

Comment:
Are these material themes ones we couldn't abstract to separate css classes?

Reply:
I just moved the theme components to src/styles/index.js, this appears to be the recommended MUI technique